docs/index.rst

```diff
@@ -120,14 +120,14 @@ See the `Cortex Fabric documentation. <https://cognitivescale.github.io/cortex-f
 Cortex Python Builders
 ------------------------
 
-The cortex-python-builders library is an add-on library for use with the base cortex-python library that helps with building actions and skills.
+The sensa-python-builders library is an add-on library for use with the base sensa-python library that helps with building actions and skills.
```

Comment:
This is more of a product question, but do we know what the plan is for these two (extension) packages? Are we going to deprecate or update (rename):
- `cortex-python-builders`
- `cortex-python-profiles` <--- I would presume this is to be functionally replaced by the Profiles SDK

Let's make sure to get a backlog issue to address these packages?